lib/presentation/utils/money_format.dart

```diff
@@ -14,7 +14,7 @@ class MoneyFormat {
   }
 
   static String formatPositiveAmountCurrency(double amount, String currency) {
-    if (amount.abs() >= 50000) {
+    if (amount.abs() >= 1000) {
```

Comment:
why did you change that?

Reply:
because we decided that there would be a shortened version of the number after one thousand